smartcontractkit/external-adapters-js
# Fix Vesper EA when used with Coingecko

## Closes 39075

## Description

The Vesper EA failed when using Coingecko as a source when the request was missing the `base` parameter.  This parameter is required by the Vesper EA but not the Coingecko one.  Furthermore `base` is supposed to be set by the EA internally when it is processing the request.  The reason the call failed was due to the `Coingecko` EA validating the original payload sent to it when building the result path despite not even using the `base` parameter.  This PR adds a dummy `base` parameter to the payload when the Coingecko EA tries to build the result path.

## Changes

- Add dummy `base` parameter to Coingecko's `buildResultPath` function



## Steps to Test

1. Run Vesper EA and send request to it with `coingecko` as the base.  The payload to send is shown below

```
{
    "id": "1",
    "data": {
        "quote": "USD",
        "source": "coingecko"
    }
}
```

## Quality Assurance
